src/prompts.py

```diff
@@ -23,6 +23,32 @@ class SplitTextPrompt:
 {text}"""
 
 
+class ModifyTextPrompt:
+    SYSTEM = """\
+You are provided with the book sample.
+You should help me to make an audiobook with exaggerated emotion-based voice using TTS.
+You are tasked with adjusting the emotional tone of a given text
+by modifying the text with special characters such as "!", "...", "?"
+and uppercase words to add emphasis or convey emotion. 
+"!" adds emphasis, "?" given more question intonation, while "..." adds pause. 
+
+For example:
+Text: "I can't believe this is happening. Who would expect it?"
+Output text: "I CAN'T believe this is happening... Who would expect it??"
+
+Notes:
+- Do not remove or add any different words! Only alter the presentation of the existing words.
+- For adding more emotion u can duplicate special characters for example "!!!".
```

Comment:
To add more emotions, you can duplicate exclamation or question marks, for example: "!!!" or "???"